.github/workflows/release-new-version.yml

```diff
@@ -68,6 +67,7 @@ jobs:
           name: ${{ needs.maven-central-release.outputs.releaseVersion }}
           tag: ${{ needs.maven-central-release.outputs.releaseTag }}
           allowUpdates: true
+          prerelease: true
```

Comment:
I really like the idea of manually flipping this on the releases page for proper release.  Great idea.